playground/mocks/data/idp/idx/terminal-return-otp-only-full-location-mobile-icon-authentication.json

```diff
@@ -0,0 +1,70 @@
+{
+  "version": "1.0.0",
+  "intent":"AUTHENTICATION",
```

Comment:
The differences among the 4 new json files are the `intent` and `client.value.os`(two are `Android`, the other two are `iOS`).
The values of `client.value.os` are chose based on the assumption that we only consider os name that contains "Android" or "iOS" as mobile device (pending designer and PM review. The response from server has only a OS name, not device type), as compared to the `eu.bitwalker.userAgentUtils.DeviceType`:
```
	/**
	 * Standard desktop or laptop computer
	 */
	COMPUTER("Computer"),
	/**
	 * Mobile phone or similar small mobile device
	 */
	MOBILE("Mobile"),
	/**
	 * Small tablet type computer.
	 */
	TABLET("Tablet"),
	/**
	 * Game console like the Wii or Playstation.
	 */
	GAME_CONSOLE("Game console"),
	/**
	 * Digital media receiver like the Google TV.
	 */	
	DMR("Digital media receiver"),
	/**
	 * Miniature device like a smart watch or interactive glasses
	 */	
	WEARABLE("Wearable computer"),
	/**
	 * Other or unknow type of device.
	 */	
	UNKNOWN("Unknown");
```